### ISSUE TITLE
[3.0.0-beta2] Test not passing because of sessions

### ISSUE THREAD

**Issue by b-ced** [modxcms/revolution#15939]
### Summary
Test not passing because of sessions

![2021-12-12 11_43_43-MODX Revolution 3 0 0-beta2 » Installer](https://user-images.githubusercontent.com/74956171/145709317-0c35308b-1d78-415c-9837-596dba747d97.png)

Wasn't really sure if it came from my config but i have the following error showing

```
Warning
: Undefined array key "session_test" in
C:\wamp64\www\ced\tests\modx-3.0.0-beta2\setup\includes\test\modinstalltest.class.php
on line
163
```

### Environment
MODX Revolution 3.0.0-beta2, Apache/PHP8 (French version)

**Comment by JoshuaLuckers** [comment]
@b-ced is this a fresh install or did you upgrade? Does it help if you try it in another browser?

**Comment by b-ced** [comment]
It was a fresh install, and it looks like this error only comes in Firefox, no problem with chrome.

**Comment by Ibochkarev** [comment]
Fixed #15941
